testsuite/domain/pom.xml

```diff
@@ -177,7 +177,6 @@
                     <argLine>${jvm.args.ip.client} ${jvm.args.timeouts} ${surefire.system.args}</argLine>
 
                     <systemPropertyVariables>
-                        <jboss.options>-ea ${surefire.system.args}</jboss.options>
```

Comment:
Why?

How is -ea getting set on the PC and HC process without this?

What other values were there in ${surefire.system.args} and how are they getting set on the PC and HC process without this?